Formula/clair.rb

```diff
@@ -34,6 +34,6 @@ def install
     cp etc/"clair/config.yaml.sample", testpath
     output = shell_output("#{bin}/clair -conf #{testpath}/config.yaml.sample -mode combo 2>&1", 1)
     # requires a Postgres database
-    assert_match "service initialization failed: failed to initialize indexer: failed to create ConnPool", output
+    assert_match "error accessing \"optional/path/to/rootca\"", output
```

Comment:
That seems like an actual problem in the config

Reply:
yeah, looks like sample config contains too many things.